### DISCUSSION TITLE
How can I use?

### DISCUSSION THREAD

**Discussion by murilofernandesbr** [paypal/data-contract-template#5] [Q&A] [answered]
Hey, guys!

It's a awesome project, many thanks to the PayPal team about distributing the data contract template.

I'm in doubt, how can I install this to use in my projects?

Thank you for your attention.

**Comment by jgperrin** [comment, marked as answer]
Hey @murilofernandesbr,

As you can see, it's not code; it's purely a template. The data contract between data producers and consumers can define the agreement on the data you want to share and consume. 

My team is working on a few examples to help everybody understand its usage. Some other (future) contributors are working on tools to leverage it. 

Hope it helps,

--jg
